src/main/java/io/specto/hoverfly/junit/core/HoverflyConfig.java

```diff
@@ -86,4 +89,44 @@ public boolean isProxyLocalHost() {
         return proxyLocalHost;
     }
 
+    /**
+     * By calling this it means a remote Hoverfly will be used - not started by Java
+     *
+     * @return this
+     */
+    public HoverflyConfig useRemoteInstance() {
```

Comment:
RemoteInstance should have a known port, so the port number cannot be default to 0, instead should be set to default 8888 and 8500